src/DotNet.Status.Web/appsettings.json

```diff
@@ -24,25 +24,25 @@
         {
           "Project": "internal",
           "DefinitionPath": "\\dotnet\\arcade-validation\\dotnet-arcade-validation-official",
-          "Branches": [ "master" ],
+          "Branches": [ "main" ],
```

Comment:
Arcade Validation repo (https://github.com/dotnet/arcade-validation) will need to be on main to make this change

Reply:
I confirm that arcade-validation is on main now.